function.js

```diff
@@ -0,0 +1,154 @@
+// // Перепишите функцию, чтобы она делала то же самое, но без if, в одну строку.
+// // Сделайте два варианта функции checkAge:
+// // Используя оператор ?
+// // Используя оператор ||
+
+let checkAge(age) {
+    return (age >= 18) ? true : confirm("Родители разрешили?")
+};
+
+let checkAge(age) {
+    return (age >= 18) || confirm("Родители разрешили?")
+};
+
+let checkAge(age) ? () => true : () confirm("Родители разрешили?");
+
+
+// // Напишите функцию min(a,b), которая возвращает меньшее из чисел a и b.
+function min(a, b) {
```

Comment:
We name functıons wıth a verb ın the begınnıng

Reply:
fixed